src/components/veNft/formControls/StakingFormActionButton.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from 'antd'
+import { NetworkContext } from 'contexts/networkContext'
+import { useContext } from 'react'
+
+interface StakingFormActionButtonProps {
+  hasAdequateApproval: boolean
+  onReviewButtonClick: () => void
+}
+
+const StakingFormActionButton = ({
+  hasAdequateApproval,
+  onReviewButtonClick,
+}: StakingFormActionButtonProps) => {
+  const { userAddress, onSelectWallet } = useContext(NetworkContext)
+
+  const approve = () => {
+    return
+  }
+
+  const renderActionButton = () => {
+    if (!userAddress && onSelectWallet) {
+      return (
+        <Button
+          block
+          style={{ whiteSpace: 'pre' }}
```

Comment:
**question** what's the reasoning for this style?

Reply:
Tested, unnecessary, removed.